custom-elements.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Yeah, I'm wondering if it can be fed into Storybook to define the knobs. Doesn't appear to happen yet, at least. But if they don't, we could always import this file and loop through them.

Reply:
openwc docs say that it's possible: 
> Based on the data in custom-elements.json, we can automatically generate knobs for your stories.

`https://open-wc.org/demoing/`

Q: i'm unfamiliar with the responsibility of this file, what is it for outside of demo?